lino_extjs6/extjs6/linoweb.js

```diff
@@ -2108,6 +2107,7 @@ Lino.call_ajax_action = function(
     ,success: Lino.action_handler(panel, on_success, on_confirm)
     ,failure: Lino.ajax_error_handler(panel)
   });
+    if (panel.loadMask) panel.loadMask.hide();
```

Comment:
I don't believe that this line is good. It makes no sense to hide the loadMask immediately after sending the Ajax request, you must hide it in the success and failure handlers. Or am i missing something?